src/Sortable/SortableListener.php

```diff
@@ -264,6 +265,10 @@ public function postFlush(EventArgs $args)
                                 // Otherwise we fallback to normal object comparison
                                 if ($gr instanceof Comparable) {
                                     $matches = $gr->compareTo($value);
+                                    // According to the interface "0" is returned, if the objects match.
+                                    // However, previously a bool was expected by the listener.
+                                    // Therefore, in order to avoid a breaking change, "true" and "0" are accepted.
+                                    $matches = 0 === $matches || true === $matches;
```

Comment:
it is probably not 100% non breaking because in theory someone could have returned `1` to achieve the same result but that is contrary to the previous implementation and the PHPdoc of the interface.